<launch>
	<arg name="uav_id" value="1"/>
	<arg name="uav_name" value="uav"/>
    <arg name="flag_printf" value="true"/>
    <arg name="sim_mode" value="true"/>
    <arg name="use_rc" value="false"/>
    <arg name="use_external_attitude" value="true"/>
    <arg name="Takeoff_height" value="0.6"/>
    <arg name="Land_speed" value="0.2"/>
    <arg name="Disarm_height" value="0.2"/>
    <arg name="geo_fence/x_min" value="-30.0"/>
    <arg name="geo_fence/x_max" value="30.0"/>
    <arg name="geo_fence/y_min" value="-30.0"/>
    <arg name="geo_fence/y_max" value="30.0"/>
    <arg name="geo_fence/z_min" value="-0.5"/>
    <arg name="geo_fence/z_max" value="30"/>

    <!-- 启动 sunray_control_node -->
	<node pkg="sunray_uav_control" type="sunray_control_node" name="uav_control_node_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)" />
		<param name="uav_name" value="$(arg uav_name)" />
		<param name="flag_printf" value="$(arg flag_printf)" />
		<param name="sim_mode" value="$(arg sim_mode)" />
		<param name="use_rc" value="$(arg use_rc)" />
		<param name="use_external_attitude" value="$(arg use_external_attitude)" />
		<param name="Takeoff_height" value="$(arg Takeoff_height)" />
		<param name="Land_speed" value="$(arg Land_speed)" />
		<param name="Disarm_height" value="$(arg Disarm_height)" />
		<param name="geo_fence/x_min" value="$(arg geo_fence/x_min)" />
		<param name="geo_fence/x_max" value="$(arg geo_fence/x_max)" />
		<param name="geo_fence/y_min" value="$(arg geo_fence/y_min)" />
		<param name="geo_fence/y_max" value="$(arg geo_fence/y_max)" />
		<param name="geo_fence/z_min" value="$(arg geo_fence/z_min)" />
		<param name="geo_fence/z_max" value="$(arg geo_fence/z_max)" />


		<rosparam command="load" file="$(find sunray_uav_control)/att_estimate/config/ctrl_param_fpv.yaml" />
	</node>
</launch>

